app/themes/map/ui/app/app.js

```diff
@@ -17,8 +17,10 @@
     'ui.bootstrap',
     'ui.router',
     'ui.tinymce',
+    'cb.x2js',
     'ngToast',
-    'uiGmapgoogle-maps'
+    'uiGmapgoogle-maps',
+    'ngMessages'
```

Comment:
Same here, should be in create.module.js only, which is not overridden by map theme, so I think you only need to change the default one, and just undo these changes..